components/invitee.jsx

```diff
@@ -134,7 +134,7 @@ export default () => {
           ))}
         </tbody>
       </table>
-      {pagination.next_page && (
+      {pagination && pagination.next_page && (
```

Comment:
Totally unnecessary, and just syntactic sugar, but you can do this, and it has the same effect! ✨ 
```suggestion
      {pagination?.next_page && (
```

Reply:
Oh, that's great, actually. I love it. Thank you!

Reply:
No CI/CD right now. I think that's supposed to come with the hosting process. I hadn't thought about having it now!